launch/dynamo-run/src/lib.rs

```diff
@@ -203,6 +206,7 @@ pub async fn run(
                 );
             };
             EngineConfig::StaticFull {
+                card: Box::new(ModelDeploymentCard::with_name_only(&model_name)),
```

Comment:
Can you summarize the name-only change here? Is it because the rest of the files or full MDC will be populated elsewhere with care_store changes?